pkgs/by-name/tt/ttaenc/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "ttaenc";
+  version = "3.4.1";
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/tta/${finalAttrs.name}-${finalAttrs.version}-src.tgz";
+    sha256 = "sha256-ssnIsBWsxYZPCCoBV/LgnFEX0URTIctheOkltEi+PcY=";
+  };
+
+  makeFlags = [ "INSDIR=$(out)/bin" ];
+
+  preBuild = ''
+    # From the Makefile, with `-msse` removed, since we have those on by x86_64 by default.
+    makeFlagsArray+=(CFLAGS="-Wall -O3 -fomit-frame-pointer -funroll-loops -fforce-addr -falign-functions=4")
+  '';
+
+  postInstall = ''
+    # Copy docs
+    install -dm755 "$out/share/doc/${finalAttrs.name}"
+    install -m644 "ChangeLog-${finalAttrs.version}" README "$out/share/doc/${finalAttrs.name}"
```

Comment:
nit: these two could be merged into one `install -Dm755 -t $out/share/doc/${finalAttrs.name} ChangeLog-${finalAttrs.version} README`

Reply:
Didn't know that, looks cool! Would it keep the latter files at 0644 permission?

Reply:
Good point actually. The default `umask` should enforce 644 but it isn't explicit. In that regards yours is better.